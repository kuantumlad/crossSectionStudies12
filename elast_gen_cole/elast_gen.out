  Incident energy is    7.54600000      GeV
  Minimum accepted scattered electron energy is  0.500000000    
  Target thickness is    5.78034669E-03  r.l.
  Number of particles per event in bos bank =           3
 Electron scattering angles range from   6.0 to  60.0 degrees
  Elastic peak included
  Elastic Scattering, Date and time:Mon Sep  2 16:00:10 2019    
  Mininum photon energy =   4.99999989E-03
  Maximum photon energy =   6.72757149    
  ntries,nevent, mcall_max:              10146679        2000         153
  sig_int =   0.126055360      0.125943914    
  Number of seconds at Lum=1E34 is   1.58660436    
 Elapsed CPU time =      4.1 seconds
  ntries,nevent, mcall_max:              22395426        4000         153
  sig_int =   0.114223622      0.114340067    
  Number of seconds at Lum=1E34 is   3.50190234    
 Elapsed CPU time =      8.9 seconds
  ntries,nevent, mcall_max:              34382124        6000         153
  sig_int =   0.111602470      0.111323796    
  Number of seconds at Lum=1E34 is   5.37622499    
 Elapsed CPU time =     13.6 seconds
  ntries,nevent, mcall_max:              47313810        8000         153
  sig_int =   0.108132780      0.108082063    
  Number of seconds at Lum=1E34 is   7.39831114    
 Elapsed CPU time =     18.6 seconds
  ntries,nevent, mcall_max:              59476960       10000         153
  sig_int =   0.107524268      0.107446030    
  Number of seconds at Lum=1E34 is   9.30022621    
 Elapsed CPU time =     23.4 seconds
  ntries,nevent, mcall_max:              71383025       12000         153
  sig_int =   0.107508197      0.106819704    
  Number of seconds at Lum=1E34 is   11.1619387    
 Elapsed CPU time =     28.0 seconds
  ntries,nevent, mcall_max:              78018034       14000         757
  sig_int =   0.114759408      0.114430845    
  Number of seconds at Lum=1E34 is   12.1994352    
 Elapsed CPU time =     30.6 seconds
  ntries,nevent, mcall_max:              89702886       16000         757
  sig_int =   0.114069313      0.113662519    
  Number of seconds at Lum=1E34 is   14.0265589    
 Elapsed CPU time =     35.1 seconds
  ntries,nevent, mcall_max:             102505002       18000         757
  sig_int =   0.112300761      0.111738116    
  Number of seconds at Lum=1E34 is   16.0283852    
 Elapsed CPU time =     40.1 seconds
  ntries,nevent, mcall_max:             112965404       20000         757
  sig_int =   0.113224342      0.112346597    
  Number of seconds at Lum=1E34 is   17.6640453    
 Elapsed CPU time =     44.2 seconds
  ntries,nevent, mcall_max:             124524378       22000         757
  sig_int =   0.112985730      0.112206869    
  Number of seconds at Lum=1E34 is   19.4714851    
 Elapsed CPU time =     48.7 seconds
  ntries,nevent, mcall_max:             135699442       24000         757
  sig_int =   0.113106735      0.112490632    
  Number of seconds at Lum=1E34 is   21.2188950    
 Elapsed CPU time =     53.1 seconds
  ntries,nevent, mcall_max:             145723673       26000         757
  sig_int =   0.114103369      0.113527335    
  Number of seconds at Lum=1E34 is   22.7863560    
 Elapsed CPU time =     57.0 seconds
  ntries,nevent, mcall_max:             155794427       28000         757
  sig_int =   0.114937395      0.114465334    
  Number of seconds at Lum=1E34 is   24.3610878    
 Elapsed CPU time =     60.9 seconds
  ntries,nevent, mcall_max:             167820437       30000         757
  sig_int =   0.114322491      0.114095859    
  Number of seconds at Lum=1E34 is   26.2415562    
 Elapsed CPU time =     65.6 seconds
  ntries,nevent, mcall_max:             178938403       32000         757
  sig_int =   0.114367247      0.114263266    
  Number of seconds at Lum=1E34 is   27.9800377    
 Elapsed CPU time =     69.9 seconds
  ntries,nevent, mcall_max:             188466215       34000         757
  sig_int =   0.115372062      0.115164474    
  Number of seconds at Lum=1E34 is   29.4698715    
 Elapsed CPU time =     73.7 seconds
  ntries,nevent, mcall_max:             200398787       36008         757
  sig_int =   0.114910357      0.114524148    
  Number of seconds at Lum=1E34 is   31.3357296    
 Elapsed CPU time =     78.3 seconds
  ntries,nevent, mcall_max:             211859818       38052         757
  sig_int =   0.114864051      0.114436634    
  Number of seconds at Lum=1E34 is   33.1278572    
 Elapsed CPU time =     82.8 seconds
  ntries,nevent, mcall_max:             222568504       40000         757
  sig_int =   0.114934787      0.114599310    
  Number of seconds at Lum=1E34 is   34.8023453    
 Elapsed CPU time =     86.9 seconds
  ntries,nevent, mcall_max:             234855716       42000         757
  sig_int =   0.114367701      0.113853469    
  Number of seconds at Lum=1E34 is   36.7236519    
 Elapsed CPU time =     91.7 seconds
  ntries,nevent, mcall_max:             246796250       44000         757
  sig_int =   0.114016935      0.113378577    
  Number of seconds at Lum=1E34 is   38.5907593    
 Elapsed CPU time =     96.4 seconds
  ntries,nevent, mcall_max:             256910067       46001         757
  sig_int =   0.114509463      0.113942824    
  Number of seconds at Lum=1E34 is   40.1722260    
 Elapsed CPU time =    100.3 seconds
  ntries,nevent, mcall_max:             269240977       48000         757
  sig_int =   0.114013247      0.113466166    
  Number of seconds at Lum=1E34 is   42.1003685    
 Elapsed CPU time =    105.1 seconds
  ntries,nevent, mcall_max:             280331272       50000         757
  sig_int =   0.114065342      0.113526039    
  Number of seconds at Lum=1E34 is   43.8345261    
 Elapsed CPU time =    109.5 seconds
 Elapsed CPU time =      109.5 seconds
