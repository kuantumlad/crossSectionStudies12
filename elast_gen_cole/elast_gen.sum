  Radiative elastic scattering
 Incident electron energy =  7.546 GeV
 Minimum accepted scattered electron energy is  0.500 GeV
 Target thickness = 0.0058 (r.l.)
 Each event in the bos bank contains 3 particles
 Electron scattering angles range from   6.0 to  60.0 degrees
  Elastic peak included
 Start time Wed Aug 28 08:19:01 2019    
 ntries =  16563007,  successes =   10001, mcall_max = 554
 Elapsed CPU time =     0.11 minutes
 distribution of mcall values in the inelastic tail region   1
 fraction of tail calculation spent here: 0.250
   3441    171     53     24     14      3      6      4      0      0
      1      0      0      0      0      1      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0

 distribution of mcall values in the inelastic tail region   2
 fraction of tail calculation spent here: 0.250
   1133      5      0      0      1      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0

 distribution of mcall values in the inelastic tail region   3
 fraction of tail calculation spent here: 0.050
    104      6      5      9      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      1      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      1      1      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0

 distribution of mcall values in the inelastic tail region   4
 fraction of tail calculation spent here: 0.150
    183      6      4      1      1      0      1      1      0      1
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0

 distribution of mcall values in the inelastic tail region   5
 fraction of tail calculation spent here: 0.300
    320     29     23      9      8      2      2      5      2      0
      0      0      0      1      2      1      1      2      0      1
      2      0      0      0      0      0      1      0      0      1
      0      0      0      0      0      0      0      1      1      0
      1      0      0      0      0      0      0      0      0      0
      0      0      0      1      0      0      0      0      0      0
      0      0      0      1      0      0      0      0      0      0
      0      0      0      0      0      0      1      0      0      0
      0      1      0      0      0      0      0      0      0      0
      1      0      0      0      0      0      0      0      0      3

 distribution of mcall values in the elastic peak
 fraction of total integration spent here: 0.800
   1552      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0

 number of successes per cpu second =*******
 Quality factor for the run =   2.61628
 Reference differential cross section was  0.002047
 Integrated cross section (direct)=   0.11854 micro-barns
 Integrated cross section (from successes)=   0.11860 micro-barns
 Beam time at Luminosity =1.0E34 =   8.432 seconds
