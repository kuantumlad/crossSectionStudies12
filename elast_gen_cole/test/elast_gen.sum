  Radiative elastic scattering
 Incident electron energy =  5.498 GeV
 Minimum accepted scattered electron energy is  0.500 GeV
 Target thickness = 0.0000 (r.l.)
 Each event in the bos bank contains 2 particles
 Electron scattering angles range from  14.0 to  60.0 degrees
  Elastic peak included
 Start time Fri Mar 10 16:30:43 2017    
 ntries =2356897150,  successes =   10001, mcall_max =   1
 Elapsed CPU time =    10.98 minutes
 distribution of mcall values in the inelastic tail region   1
 fraction of tail calculation spent here: 0.250
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0

 distribution of mcall values in the inelastic tail region   2
 fraction of tail calculation spent here: 0.250
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0

 distribution of mcall values in the inelastic tail region   3
 fraction of tail calculation spent here: 0.050
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0

 distribution of mcall values in the inelastic tail region   4
 fraction of tail calculation spent here: 0.150
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0

 distribution of mcall values in the inelastic tail region   5
 fraction of tail calculation spent here: 0.300
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0

 distribution of mcall values in the elastic peak
 fraction of total integration spent here: 1.000
  10001      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0
      0      0      0      0      0      0      0      0      0      0

 number of successes per cpu second = 15.183
 Quality factor for the run =  15.18270
 Reference differential cross section was  0.020109
 Integrated cross section (direct)=   0.00089 micro-barns
 Integrated cross section (from successes)=   0.00089 micro-barns
 Beam time at Luminosity =1.0E34 =1120.772 seconds
