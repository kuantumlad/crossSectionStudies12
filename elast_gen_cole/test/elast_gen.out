  Incident energy is    5.49800014      GeV
  Minimum accepted scattered electron energy is  0.500000000    
  Target thickness is    0.00000000      r.l.
  Number of particles per event in bos bank =           2
 Electron scattering angles range from  14.0 to  60.0 degrees
  Elastic peak included
  Elastic Scattering, Date and time:Fri Mar 10 16:30:43 2017    
  Mininum photon energy =   2.99999993E-02
  Maximum photon energy =   4.18290138    
  ntries,nevent, mcall_max:              95792639         400           1
  sig_int =    8.78168095E-04   8.92258948E-04
  Number of seconds at Lum=1E34 is   45.5493660    
 Elapsed CPU time =     26.9 seconds
  ntries,nevent, mcall_max:             184311627         800           1
  sig_int =    9.12824064E-04   8.92297598E-04
  Number of seconds at Lum=1E34 is   87.6401062    
 Elapsed CPU time =     51.7 seconds
  ntries,nevent, mcall_max:             283757001        1200           1
  sig_int =    8.89374118E-04   8.92310112E-04
  Number of seconds at Lum=1E34 is   134.926346    
 Elapsed CPU time =     79.7 seconds
  ntries,nevent, mcall_max:             381490235        1600           1
  sig_int =    8.82036053E-04   8.92319775E-04
  Number of seconds at Lum=1E34 is   181.398483    
 Elapsed CPU time =    107.1 seconds
  ntries,nevent, mcall_max:             475219705        2000           1
  sig_int =    8.85085727E-04   8.92301556E-04
  Number of seconds at Lum=1E34 is   225.966797    
 Elapsed CPU time =    133.4 seconds
  ntries,nevent, mcall_max:             561018434        2400           1
  sig_int =    8.99671402E-04   8.92316573E-04
  Number of seconds at Lum=1E34 is   266.764069    
 Elapsed CPU time =    157.3 seconds
  ntries,nevent, mcall_max:             659583278        2800           1
  sig_int =    8.92767101E-04   8.92340846E-04
  Number of seconds at Lum=1E34 is   313.631622    
 Elapsed CPU time =    184.7 seconds
  ntries,nevent, mcall_max:             749688548        3200           1
  sig_int =    8.97674588E-04   8.92355165E-04
  Number of seconds at Lum=1E34 is   356.476624    
 Elapsed CPU time =    209.9 seconds
  ntries,nevent, mcall_max:             844599544        3600           1
  sig_int =    8.96399200E-04   8.92357551E-04
  Number of seconds at Lum=1E34 is   401.606781    
 Elapsed CPU time =    236.5 seconds
  ntries,nevent, mcall_max:             938174490        4000           1
  sig_int =    8.96656711E-04   8.92344106E-04
  Number of seconds at Lum=1E34 is   446.101593    
 Elapsed CPU time =    262.7 seconds
  ntries,nevent, mcall_max:            1033877448        4400           1
  sig_int =    8.95021425E-04   8.92346085E-04
  Number of seconds at Lum=1E34 is   491.608337    
 Elapsed CPU time =    289.6 seconds
  ntries,nevent, mcall_max:            1125389716        4800           1
  sig_int =    8.96990998E-04   8.92336480E-04
  Number of seconds at Lum=1E34 is   535.122437    
 Elapsed CPU time =    315.2 seconds
  ntries,nevent, mcall_max:            1221880307        5200           1
  sig_int =    8.95002973E-04   8.92344397E-04
  Number of seconds at Lum=1E34 is   581.003662    
 Elapsed CPU time =    342.2 seconds
  ntries,nevent, mcall_max:            1320623310        5600           1
  sig_int =    8.91782343E-04   8.92339740E-04
  Number of seconds at Lum=1E34 is   627.955933    
 Elapsed CPU time =    369.8 seconds
  ntries,nevent, mcall_max:            1418411980        6000           1
  sig_int =    8.89607996E-04   8.92332930E-04
  Number of seconds at Lum=1E34 is   674.454346    
 Elapsed CPU time =    397.3 seconds
  ntries,nevent, mcall_max:            1509822865        6400           1
  sig_int =    8.91464006E-04   8.92335724E-04
  Number of seconds at Lum=1E34 is   717.920166    
 Elapsed CPU time =    422.9 seconds
  ntries,nevent, mcall_max:            1603135226        6800           1
  sig_int =    8.92048643E-04   8.92333337E-04
  Number of seconds at Lum=1E34 is   762.290283    
 Elapsed CPU time =    449.0 seconds
  ntries,nevent, mcall_max:            1694169611        7200           1
  sig_int =    8.93769262E-04   8.92334385E-04
  Number of seconds at Lum=1E34 is   805.577026    
 Elapsed CPU time =    474.6 seconds
  ntries,nevent, mcall_max:            1791853380        7600           1
  sig_int =    8.91991949E-04   8.92329903E-04
  Number of seconds at Lum=1E34 is   852.025635    
 Elapsed CPU time =    502.1 seconds
  ntries,nevent, mcall_max:            1880703550        8000           1
  sig_int =    8.94580677E-04   8.92328215E-04
  Number of seconds at Lum=1E34 is   894.273682    
 Elapsed CPU time =    527.0 seconds
  ntries,nevent, mcall_max:            1972455897        8400           1
  sig_int =    8.95615900E-04   8.92324664E-04
  Number of seconds at Lum=1E34 is   937.902039    
 Elapsed CPU time =    552.9 seconds
  ntries,nevent, mcall_max:            2066337424        8800           1
  sig_int =    8.95635400E-04   8.92330892E-04
  Number of seconds at Lum=1E34 is   982.542664    
 Elapsed CPU time =    579.8 seconds
  ntries,nevent, mcall_max:            2160002761        9200           1
  sig_int =    8.95742793E-04   8.92334909E-04
  Number of seconds at Lum=1E34 is   1027.08044    
 Elapsed CPU time =    606.3 seconds
  ntries,nevent, mcall_max:            2253266335        9600           1
  sig_int =    8.96001002E-04   8.92335374E-04
  Number of seconds at Lum=1E34 is   1071.42737    
 Elapsed CPU time =    631.0 seconds
  ntries,nevent, mcall_max:            2356637470       10000           1
  sig_int =    8.92394804E-04   8.92331824E-04
  Number of seconds at Lum=1E34 is   1120.58020    
 Elapsed CPU time =    658.6 seconds
 Elapsed CPU time =      658.7 seconds
